cla_public/templates/moj_template_base.jinja

```diff
@@ -31,153 +41,164 @@
+  <link rel="apple-touch-icon" sizes="152x152" href="{{ static('images/govuk-apple-touch-icon-152x152.png') }}">
+  <link rel="apple-touch-icon" href="{{ static('images/govuk-apple-touch-icon.png') }}">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta property="og:image" content="{{ static('images/opengraph-image.png') }}">
+  <meta name="theme-color" content="#0b0c0c"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+
+  <!--[if gt IE 8]><!-->
+  <link href="{{ static('stylesheets/moj-template.css') }}" media="screen" rel="stylesheet" type="text/css"/>
+  <!--<![endif]-->
+  <!--[if IE 6]><link href="{{ static('stylesheets/moj-template-ie6.css') }}" media="screen" rel="stylesheet" type="text/css" /><![endif]-->
+  <!--[if IE 7]><link href="{{ static('stylesheets/moj-template-ie7.css') }}" media="screen" rel="stylesheet" type="text/css" /><![endif]-->
+  <!--[if IE 8]><link href="{{ static('stylesheets/moj-template-ie8.css') }}" media="screen" rel="stylesheet" type="text/css" /><![endif]-->
+
+  {% block stylesheets %}{% endblock %}
+  {% block head %}{% endblock %}
+
+</head>
+
+<body class="govuk-template__body
   {% block body_classes %}{{ product_type }} {{ phase }} {{ 'cla-' + config["CLA_ENV"] }} {% endblock %}">
-    <script type="text/javascript">document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
-
-    <div class="laa-flee-button-container
-      {% if is_quick_exit_enabled(session) %}
-        govuk-visually-hidden
-      {% endif %}
-      ">
-      <p class="govuk-body">
-        ({{ _('To leave the page quickly, press the escape key.') }})
-      </p>
-    </div>
-    <div id="skiplink-container">
-      <div>
-        <a href="#main-content" class="govuk-skip-link">{{ _('Skip to main content') }}</a>
-      </div>
-    </div>
-
-    <div id="cookie-message" class="gem-c-cookie-banner" data-module="cookie-banner" role="region" aria-label="cookie banner">
-      {% block cookie_message %} {% endblock %}
-    </div>
-    <!--end global-cookie-message-->
 
-    
-    <header class="govuk-header " role="banner" data-module="govuk-header">
-      <div class="govuk-header__container govuk-width-container {% block error_class %}{% endblock %}">
-        <div class="govuk-header__logo">
-          <a href="https://www.gov.uk/" class="govuk-header__link govuk-header__link--homepage">
+<script
+  type="text/javascript">document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
+
+<div class="laa-flee-button-container {% if is_quick_exit_enabled(session) %} govuk-visually-hidden {% endif %}"></div>
+<div id="skiplink-container">
+  <div>
+    <a href="#main-content" class="govuk-skip-link">{{ _('Skip to main content') }}</a>
+    <a href="https://bbc.co.uk/weather/" class="govuk-skip-link govuk-js-exit-this-page-skiplink"
```

Comment:
This will show up everywhere in the website even if they're not on the domestic abuse parts of the site.

Reply:
This can be fixed by using the following implementation

`{% if is_quick_exit_enabled(session) %}
    <a href="https://bbc.co.uk/weather/" class="govuk-skip-link govuk-js-exit-this-page-skiplink "
       rel="nofollow noreferrer" data-module="govuk-skip-link">Exit this page</a>
    {% endif %}`
    
    Many thanks 😄